src/plugins/discover/server/saved_objects/search.ts

```diff
@@ -18,9 +18,9 @@
  */
 
 import { SavedObjectsType } from 'kibana/server';
-import { searchSavedObjectTypeMigrations } from './search_migrations';
+import { searchMigrations } from './search_migrations';
 
-export const searchSavedObjectType: SavedObjectsType = {
+export const search: SavedObjectsType = {
```

Comment:
//nit

IMO the name is waaay too generic ;-)

Reply:
@lizozom i do agree, but I've named it what I was thought is the convention, and it's imported just once to register :) https://github.com/elastic/kibana/blob/master/src/core/CONVENTIONS.md#saved-objects-types
should I go back or celebrate  minus of 15 characters here, or should I name it `s` to celebrate 20chars?

Reply:
@lizozom not that I rolled I've renamed this, since the less generic, the better it's searchable